Index: client/src/infos.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const serverRouting = \"http://192.168.1.20:5000\";\r\n\r\nfunction RouteToServer(path: string){\r\n    return serverRouting + path;\r\n}\r\nexport default RouteToServer;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/client/src/infos.ts b/client/src/infos.ts
--- a/client/src/infos.ts	(revision f49ac4fd76cb3b2d2ae9fe9d61d6f28dd4c82a2d)
+++ b/client/src/infos.ts	(date 1730488798694)
@@ -1,4 +1,4 @@
-const serverRouting = "http://192.168.1.20:5000";
+const serverRouting = "http://localhost:5000";
 
 function RouteToServer(path: string){
     return serverRouting + path;
